packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -21,10 +21,14 @@
   "engines": {
```

Comment:
Could you run ./scripts/reformat-with-prettier.js at the top-level of the salesforcedx-vscode folder?

Reply:
Done.